### PR TITLE
Make some dev stack improvements

### PR DESCRIPTION
**Pull request by guicheffer**
cannot work here since we have no `yarn` package and a hook for linting our javascript 👀 

## Description
Any commit can have a precheck before with a `lint` and a unit tests being run

## Motivation, Consideration and Context
- Now with a `lint` and `jest` we guarantee a single commit once we have many lines of code on the project (that avoids CircleCI to be run after a little change on code)
- With `make help` we can see exactly what to run within many commands

## Screenshots (if appropriate):
image
image

### CODE REVIEW

File: browser.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+(() => {
```
Comment (alissonperez) [inline browser.js]: What do you think about not add this file to versioning, it's not better to build it with CI and upload to our package manger when we create a new version?

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+	@echo
+	@echo "✍🏽  Please use 'make <target>' where <target> is one of the commands below:"
+	@echo
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e "s/\\$$//" | sed -e "s/##//"
+	@echo
+
+lint: ## lints javascript lines
+	$(PKG) lint
+
+prepare: lint test ## checks if a commit can be created
```
Comment (alissonperez) [inline Makefile]: Nice target, I suggest just to remove this comment, it not necessary needs to be associated with commit, it's a target that just runs lint and tests like a shortcut for this targets.

File: browser.js (hunk 1 above)
Comment (guicheffer) [reply browser.js]: sorry bro, didn't want to add this file

File: Makefile (hunk 2 above)
Comment (guicheffer) [reply Makefile]: yep you're right

**Comment by guicheffer** [comment]
pls review it again @alissonperez

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+## Changing the code-base
+
+Generally speaking, you should fork this repository, make changes in your
+own fork, and then submit a pull-request. All new code should have associated unit tests that validate implemented features and the presence or lack of defects.
+
+Additionally, the code should follow any stylistic and architectural guidelines
+prescribed by the project. In the absence of such guidelines, mimic the styles
+and patterns in the existing code-base.
+
+## How to merge
```
Comment (alissonperez) [inline CONTRIBUTING.md]: Please, remove this! =) Thaaaannnk you!! Awesome!

**Comment by guicheffer** [comment]
@alissonperez pls https://github.com/alissonperez/deepicker/pull/7/commits/d816b51dbbde329bccad32ebd20d3bc2011963ab

next steps are (in terms of documentation):
- a nice `README.md` pointing those _Markdown_ files made on this PR)
- an awesome website (homepage) for **deepicker** 🖤

**Comment by guicheffer** [comment]
❤️

**Pull request merged**